src/test/functional/features/joint-hub-page.feature

```diff
@@ -49,6 +49,15 @@ Feature: Joint hub page
     And the page should include "You can view and download your 'certificate of entitlement for a conditional order'."
 
     Given I set the case state to "AwaitingPronouncement"
+    When I click "Sign out"
+    And I login with applicant "1"
+    Then the page should include "You and your husband have applied for a 'conditional order'."
+    Then the page should include "The court will check your application and send it to a judge."
+    When I click "Sign out"
+    And I login with applicant "2"
+    Then the page should include "You and your wife have applied for a 'conditional order'."
+    Then the page should include "The court will check your application and send it to a judge."
```

Comment:
couldn't we check for app2 first and then app1 so we don't need to sign out and log back in twice?